lib/ansible/modules/network/nxos/nxos_ip_interface.py

```diff
@@ -105,6 +111,16 @@
     addr: 20.20.20.20
     mask: 24
 
+- name: Ensure ipv4 address is configured with tag
```

Comment:
This seems to be the description of the example above, and vice versa

Reply:
Modified